test/e2e-image/Makefile

```diff
@@ -1,9 +1,9 @@
-IMAGE=nginx-ingress-controller:e2e
-
-.PHONY: all container getbins clean
+.PHONY: all container clean
 
+.PHONE: all
```

Comment:
you meant `.PHONY`?

Reply:
actually you don't need these anymore, you are declaring these following three targets as PHONY above

Reply:
fixed